File: src/led/LEDStrip.java
package led;
import java.awt.Color;
import java.util.ArrayList;

public class LEDStrip {
	ArrayList<Color> mStrip = new ArrayList<Color>(300);
	
	public LEDStrip(int _ledCount) {
		mStrip = new ArrayList<Color>(_ledCount);
		
		for (int i = 0; i < LEDStripManager.LED_COUNT; i++) {
			mStrip.add(Color.BLACK);
		}
	}
	
	public Color getColorPyPixel(int _id) {
		return mStrip.get(_id);
	}
}

File: src/led/LEDStrip.kt
package led

import java.awt.Color

class LEDStrip(_ledCount: Int) {
    var mStrip = ArrayList<Color?>(300)
    fun getColorPyPixel(_id: Int): Color? {
        return mStrip[_id]
    }

    init {
        mStrip = ArrayList(_ledCount)
        for (i in 0 until LEDStripManager.Companion.LED_COUNT) {
            mStrip.add(Color.BLACK)
        }
    }
}